R/standard-path.R

```diff
@@ -0,0 +1,39 @@
+
+#' Standard containers typically available in a PBPK simulation
+#'
+#' @include enum.R
+#' @export
+StandardContainer <- enum(c(

```

Comment:
Bunch of standard containers, path and list of default molecule parameter (all the one listed by @Yuri05 )
Question
PascaleCase or camelCase?

e.g. StandardPath@age or StandardPath@Age ?